acceptance/README.md

```diff
@@ -1,257 +1,467 @@
+parses that string.
+
+### hosts.yaml
+The rake tasks that run acceptance will by default create a hosts file and
+populate it using [beaker-hostgenerator][] using either the `HOSTS` environment
+variable or the default host string (currently `redhat7-64ma-windows2012r2-64a`).
+The automation assumes you are using the vmpooler hypervisor and an internal
+pooling api, which is only available to Puppet, Inc. employees. If you want to
+customize the hypervisor or the pooling api, you'll need to generate your own
+hosts file. You must pass in a valid host string to the `beaker-hostgenerator`
+command. See the previous section for more information on how to construct a
+valid host string.
+
+To customize the hypervisor, pass in `--hypervisor {hypervisor name}`. To set
+the pooling api, use `--global-config pooling_api={custom api}`. Only the
+vmpooler hypervisor uses the pooling api.
+
+The host string that is passed in is the same that you would use with the
+`HOSTS` environment variable. See the [previous section](##customizing-test-targets) on how to format this string.
+
+For example, if you were to run this command:
+```
+bundle exec beaker-hostgenerator redhat7-64ma-windows2012r2-64a --disable-default-role --osinfo-version 1 --hypervisor vmpooler --global-config pooling_api=http://customvmpooler/ > hosts.yaml
+```
+You would generate a file called `hosts.yaml` that contains something like this:
+```
+---
+HOSTS:
+  redhat7-64-1:
+    platform: el-7-x86_64
+    packaging_platform: el-7-x86_64
+    template: redhat-7-x86_64
+    hypervisor: vmpooler
+    roles:
+    - master
+    - agent
+  windows2012r2-64-1:
+    platform: windows-2012r2-64
+    packaging_platform: windows-2012-x64
+    ruby_arch: x64
+    template: win-2012r2-x86_64
+    hypervisor: vmpooler
+    roles:
+    - agent
+CONFIG:
+  nfs_server: none
+  consoleport: 443
+  pooling_api: http://customvmpooler/
+```
+To have the automation recognize and use your custom hosts file, you'll need to
+set the `HOSTS` environment variable to the hosts file. In the above example, we
+called this file `hosts.yaml`, so we will use `HOSTS=hosts.yaml` when running
+all future beaker commands or rake tasks to run acceptance tests.
+
+### Hypervisor Options
+The hypervisor dictates where you will be running the acceptance tests. The beaker
+hypervisors take care of basic host setup so that you will have a consistent
+host environment across every test run. You can find more details on the different
+hypervisor options in [the beaker repo](https://github.com/puppetlabs/beaker/blob/master/docs/how_to/hypervisors/README.md).
+
+Here, we will focus on vmpooler and docker, as those are the two we use most
+often internally. If you use a hypervisor other than abs, vagrant, vmpooler, or
+docker, you'll have to add the gem to that hypervisor to `Gemfile.local` and run
+`bundle update` to install the new gems. You also have the ability to run tests
+on a static host, which I will cover briefly.
```

Comment:
nit- remove "which I will cover briefly." They'll see the section if they keep reading or scroll to the next sub-heading.